conduit/data/datamodules/audio/base.py

```diff
@@ -60,14 +60,12 @@ def test_transforms(self, transform: Optional[AudioTform]) -> None:  # type: ign
             self._test_data.transform = transform
 
     @property
-    def _default_train_transforms(self) -> A.Compose:
-        transform_ls: List[AlbumentationsTform] = [torchaudio.transforms.Spectrogram()]
-        return A.Compose(transform_ls)
+    def _default_train_transforms(self) -> T.Spectrogram:
```

Comment:
hmmm, should the return type be `nn.Module`?

Reply:
No, it could be AudioTForm but prefer narrower return types.